integration/test/performancetest/demo.sh

```diff
@@ -0,0 +1,14 @@
+for commitCount in {0..12}
```

Comment:
Do we need this in the commit?

Reply:
Oh im sorry I though that was in my gitignore